website/docs/d/cis_dns_record.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+
+layout: "ibm"
+page_title: "IBM : Cloud Internet Service DNS Record"
+sidebar_current: "docs-ibm-datasources-cis-dns-record"
```

Comment:
naming convention should be inline with the datasource name
Please change `sidebar_current ` to `docs-ibm-datasources-cis-dns-records` and the file name as well to `cis_dns_records.html.markdown`

Reply:
done